game-core/src/main/java/games/strategy/engine/framework/startup/ui/pbem/ForumPosterEditorViewModel.java

```diff
@@ -0,0 +1,113 @@
+package games.strategy.engine.framework.startup.ui.pbem;
```

Comment:
The only real new logic in this class is to default the forum selection, otherwise all the logic here is extracted from the view class. Notice as well that the button clicks are not here either, those are more properly model methods and we delegate those actions to other components. In this way the view model is responsible for modeling view state, the logic behind view state updates, and for triggering model interactions and forwarding the needed data from view to the model (for example when doing a test post, we trigger the test post method call and we pass the forum selection and forum id).